### DISCUSSION TITLE
아~~~~~~~23일 노래 당첨이요~~~~~~

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#145] [잡담]
https://github.com/4Moyede/HexatonClass-01/blob/S_JB/Mentee/HJB/02/HW_Calculator.py

```python
a = Calculator( int(input(\"피연산자1 : \")), input(\"연산기호 : \") , int(input(\"피연산자2 : \")) )
```

@JaebeomHan 노래 당첨~~~~~~

**Comment by ghost** [comment]
ㅋㅋㅋㅋㅋㅋ이의 있습니다... cal이라는 이름으로 인스턴스를 생성할까 생각해봤는데 이 계산기에서 생성한 인스턴스는 하나뿐이어서 혼동의 여지도 없고 짧은 객체 이름은 오히려 코드 짜는데 편리하다는 생각이 들어서 그렇게 해봤습니다

**Reply by yoomambo** [reply]
> ㅋㅋㅋㅋㅋㅋ이의 있습니다... cal이라는 이름으로 인스턴스를 생성할까 생각해봤는데 이 계산기에서 생성한 인스턴스는 하나뿐이어서 혼동의 여지도 없고 짧은 객체 이름은 오히려 코드 짜는데 편리하다는 생각이 들어서 그렇게 해봤습니다

이의가 충분한 논리가 없다고 생각합니다 ^_^

**Comment by dlrudgnsdlek** [comment]
@JaebeomHan  신청곡 받나여? 임재범 고해 듣고시퍼여

**Comment by dev-LeoKang** [comment]
@JaebeomHan https://youtu.be/cuaKI3FzbpA

**Comment by yoomambo** [comment]
@NA-DK 

```python
c = calculate(num_two)
```

노래당첨~~~

**Reply by ghost** [reply]
zzzzzzzzzzzz

**Comment by dev-LeoKang** [comment]
@JaebeomHan @NA-DK  https://youtu.be/lCBMDQiz594
